WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -75,7 +75,7 @@ @implementation JetpackSettingsViewController
 
 - (instancetype)initWithBlog:(Blog *)blog
 {
-    self = [super init];
+    self = [super initWithNibName:nil bundle:nil];
```

Comment:
Curious about this change. I wouldn't think it calling `initWithNibName:bundle:` would make any practical difference.